inspirehep/modules/relations/builders/conferences/builder.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+from inspirehep.modules.relations.graph_representation import GraphModelBuilder
+from inspirehep.modules.relations.model.nodes import ConferenceNode
+from inspirehep.modules.relations.model_updaters import (
+    set_property_of_central_node,
+    set_uid
+    )
+
+
+conferences = GraphModelBuilder(central_node_type=ConferenceNode)
+
+@conferences.element_processor('control_number')
+def recid(model, element):
```

Comment:
As this is repeated for all collections, would it make sense to add it to the `common/builder.py` ?